std/bitmanip.d

```diff
@@ -300,7 +300,7 @@ The example above creates a tagged reference to an Object in the struct A. This
 as $(D taggedPointer), except the first argument which must be a class type instead of a pointer type.
 
 $(RED Warning: Don't use $(D taggedClassRef) with references to garbage collected objects, as it will result in
-undefined behaviour. See $(LINK http://dlang.org/garbage.html) for details.)
+undefined behaviour. See $(LINK ../spec/garbage.html) for details.)
```

Comment:
Ugh, right. This is a recurring problem with Ddox. See also: <https://github.com/D-Programming-Language/dlang.org/pull/1082#issuecomment-142636522>.

That should be fixed properly, of course. But that's kinda hard, so I think we have two options right now:
1) Change to absolute URLs and accept that they won't work properly in a local context (releases, git forks).
2) Stay with relative URLs and accept adding some more broken links to the /library/ pages.

I'm for 2. /library/ is still a "preview", so I put more weight on /phobos/ pages working offline, too.